src/react-components/ory/sections/oidc-section.tsx

```diff
@@ -1,4 +1,6 @@
 import { filterNodesByGroups } from "@ory/integrations/ui"
+import { JSX } from "react"
```

Comment:
i don't think these imports are necessary. they are inferred by vite through tsconfig https://github.com/ory/elements/blob/main/tsconfig.json#L20

Reply:
I get warnings in the editor, that's why I added it. But yeah, seems like it worked before.